src/components/lists/items/ImportLogListItem.scss

```diff
@@ -0,0 +1,61 @@
+.ImportLogListItem {
+    position: relative;
+
+    &::after {
+        content: "";
+        display: block;
+        clear: both;
+    }
+
+    span {
+        font-size: 1.4em;
+        line-height: 1.5em;
+        display: block;
+        overflow: hidden;
+        text-overflow: ellipsis;
+    }
+
+    .ImportLogListItem-col {
+        @include small-screen {
```

Comment:
I don't mind keeping this, but please spend no time on tweaking small-screen layouts of this since the import section is currently hidden on mobile.